plugin/src/main/scala/tut/Plugin.scala

```diff
@@ -73,13 +72,15 @@ object Plugin extends sbt.Plugin {
       libraryDependencies += "org.tpolecat" %% "tut-core" % BuildInfo.version % "test",
       tutSourceDirectory := sourceDirectory.value / "main" / "tut",
       tutTargetDirectory := crossTarget.value / "tut",
-      watchSources <++= tutSourceDirectory map { path => (path ** "*.md").get },
       tutScalacOptions := {
         val testOptions = scalacOptions.in(Test).value
         val unwantedOptions = Set("-Ywarn-unused-import")
         testOptions.filterNot(unwantedOptions)
       },
       tutNameFilter := """.*\.(md|markdown|txt|htm|html)""".r,
+      watchSources in Defaults.ConfigGlobal ++= (tutSourceDirectory.value ** new NameFilter {
```

Comment:
It's probably not necessary, although it will not have unintended consequences (that I know of). 

Here's my reasoning, which might very well be incorrect:
`watchSources` is not a scoped task, and putting in any configuration other than the global one will compile but have no effect. Putting it explicitly in the global configuration prevents tricky situations like:

```scala
def mySettings: Seq[Setting[_]] = Seq(watchSources := ???)

inConfig(Compile)(mySettings)
```

You believe you've set `watchSources` in the global configuration, but it's called in such a way that it's actually been set in the `Compile` one and will be ignored. Must head-scratching will ensue.